### ISSUE TITLE
Support colors in MacOS Terminal.app

### ISSUE THREAD

**Issue by brandonchinn178** [BurntSushi/toml-test#111]
MacOS Terminal.app does not support true colors, which causes my black background terminal to show "FAIL" as basically invisible. Maybe `toml-test` could inspect `$TERM` and use `zli.Red` if the terminal doesn't support true color? Or maybe use `Color256` with 224, which seems close enough? (`#ffd7d7`; https://www.ditig.com/256-colors-cheat-sheet)

**Comment by arp242** [comment]
I figured everything supports true colour these days, but I guess not.

I changed it to the suggested 256color "224" now, and I also changed the foreground to use 256color rather than 16color, so if one doesn't work the other also doesn't work.

Maybe we don't need to set the foreground colour at all, but as someone who uses a white background, programs setting an unreadable colour combination is somewhat common, so I tend to set both to ensure a readable combination. The "washed out red" should work on most, so probably not really needed. I'm not entirely sure how well it looks on a dark background actually.
